### PR TITLE
add episode 2 normal themes

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/s01e02.json [hunk 1]
```diff
@@ -33,6 +33,62 @@
...
+        "NumericValue": 4,
+        "StringValue": "denial",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-d04669f3",
+        "CodeType": "Normal",
+        "DisplayText": "rumour misinfo",
+        "NumericValue": 5,
+        "StringValue": "rumour_misinfo",
```
Comment (as2388) [inline code_schemes/s01e02.json]: (over on the ke projects this is rumour/stigma/misinfo)

**Review by as2388 — APPROVED** [review]

File: code_schemes/s01e02.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/s01e02.json]: Thanks, @as2388. @lukechurch should we update this theme to match urban-ke - rumour/stigma/misinfo?

Comment (lukechurch) [reply code_schemes/s01e02.json]: sure

**Review by lukechurch — APPROVED** [review]
OK for now, thanks @IsaackMwenda

**Pull request merged**